src/ssl.c

```diff
@@ -23339,7 +23350,19 @@ int wolfSSL_X509_cmp(const WOLFSSL_X509 *a, const WOLFSSL_X509 *b)
 
 #ifndef NO_CERTS
 #if defined(OPENSSL_EXTRA) || defined(OPENSSL_EXTRA_X509_SMALL)
-
+#if defined(WOLFSSL_QT)
```

Comment:
Use OPENSSL_VERSION_NUMBER for switch here as well instead of WOLFSSL_QT.

Reply:
Guard by OPENSSL_VERSION_NUMBER